codemeta.json

```diff
@@ -208,9 +268,8 @@
         "regions>=0.5.0",
         "pyyaml>=5.3",
         "click>=7.0",
-        "pydantic>=2.5",
+        "pydantic>=2.5.0",
         "iminuit>=2.8.0",
         "matplotlib>=3.4"
-    ],
-    "referencePublication": "https://doi.org/10.1051/0004-6361/202346488"
+    ]
```

Comment:
```suggestion
    ],
    "programmingLanguage": "Python",
    "referencePublication": "https://doi.org/10.1051/0004-6361/202346488"    
```

Reply:
This will require changes in `dev/codemeta.py` as the file automatically generated.